README.md

```diff
@@ -664,16 +664,34 @@ Other Style Guides
     ```
 
 
-  - [8.4](#8.4) <a name='8.4'></a> If your function only takes a single argument, feel free to omit the parentheses.
+  - [8.4](#8.4) <a name='8.4'></a> If your function takes a single argument and doesn’t use braces, omit the parentheses.
```

Comment:
let's add "Otherwise, always include parentheses around arguments." to communicate that it's not up to the individual developer.